### PR TITLE
[COD-46] Add i18n for backend

### PR DESCRIPTION
**Pull request by Set27**
## [Playbook guide](https://legolas.atlassian.net/wiki/spaces/SD/pages/5537793/I18n+for+backend?atlOrigin=eyJpIjoiOThmYTI0ZTkwMDg1NDNhZDk2MTY5ZmYzYzQxMDEwOGYiLCJwIjoiaiJ9)

# Frontend

img-2023-04-03-13-47-12
img-2023-04-03-13-46-45

# Backend
1. Add method set_locale for application controller
2. Setup en.yml and ru.yml configs for i18n
3. Add tests

### CODE REVIEW

File: backend/app/controllers/application_controller.rb [hunk 1]
```diff
@@ -20,4 +20,12 @@ def authenticate!
   rescue JWT::ExpiredSignature
     head :unauthorized
   end
+
+  def set_locale
+    I18n.locale = extract_locale_from_accept_language_header
```
Comment (npupko) [inline backend/app/controllers/application_controller.rb:25]: In order to make sure we will not break the app with weird/wrong values from header we could fallback to default:

```ruby
  def set_locale
    I18n.locale = valid_locale || I18n.default_locale
  end

  def valid_locale
    locale = request.env['HTTP_ACCEPT_LANGUAGE'].to_s.scan(/^[a-z]{2}/).first
    locale if I18n.available_locales.map(&:to_s).include?(locale)
  end
```

File: backend/spec/requests/api/v1/groups_spec.rb [hunk 2]
```diff
@@ -10,6 +10,7 @@
   let(:headers) { auth_header_for(user) }
 
   it 'creates a group' do
+    binding
```
Comment (npupko) [inline backend/spec/requests/api/v1/groups_spec.rb:13]: Watafak?

**Review by npupko — APPROVED** [review]
LGTM

**Review by SilaEwGenii — APPROVED** [review]
lgtm

**Pull request merged**